### PR TITLE
Update INSTALL.rst

### PR DESCRIPTION
**Pull request by mananpal1997**

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -245,6 +245,8 @@ Launch the data loading scripts in a separate shell:
 
 .. code-block:: console
 
+   $ cd ~/src
+   $ git clone https://github.com/inveniosoftware/invenio-openaire.git
    $ cd ~/src/zenodo
    $ workon zenodo
```
Comment (satwikkansal) [inline INSTALL.rst]: A minor suggestion, `cd zenodo` instead of `cd ~/src/zenodo` in L250 :)

**Comment by satwikkansal** [comment]
Please change the commit message according to https://github.com/zenodo/zenodo/pull/973#issuecomment-283006445

**Comment by mananpal1997** [comment]
updated :smile:

**Comment by krzysztof** [comment]
Thanks @mananpal1997 :+1:

**Pull request merged**